package.json

```diff
@@ -57,30 +57,32 @@
 		"@cosmjs/tendermint-rpc": "~0.30.0",
```

Comment:
Shouldn't we be bumping CosmJS dependencies as well?

Reply:
No they are not compatible with react native which is a requirement for credo. That's the reason we have release 2.x